judge/migrations/0154_blog_post_org.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.24 on 2021-09-03 08:56
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('judge', '0153_new_roles'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='blogpost',
+            options={'permissions': (('edit_all_post', 'Edit all posts'), ('edit_organization_post', 'Edit organization posts')), 'verbose_name': 'blog post', 'verbose_name_plural': 'blog posts'},
+        ),
+        migrations.AddField(
+            model_name='blogpost',
+            name='global_post',
+            field=models.BooleanField(default=False, help_text='Display this blog post at the homepage.', verbose_name='global post'),
```

Comment:
The `default=False` make all old blog became hidden to users, so try to run a python script here.